contrib/test/integration/swap.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Obtain current state of swap
+  command: swapon --noheadings --show NAME
+  register: swapon
+
+- name: Setup swap if none already, to prevent kernel firing off the OOM killer
```

Comment:
See the ``when`` condition at the end.  This entire block is skipped if any swap is enabled on the system.